Story/Friday13th/CoreFriday13th.cs

```diff
@@ -0,0 +1,825 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class CoreFriday13th
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new();
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        Core.RunCore();
+
+        Core.SetOptions(false);
+    }
+
+    public void DoAll()
+    {
+        Skullpunch();
+        Lowe();
+        Saloonfront();
+        Firehouse();
+        Sleezter();
+        Neverhub();
+        Battledoom();
+        Wormhole();
+        Crownsreachfxiii();
+        Gonnagetcha();
+        Greymoor();
+        Puzzlebox();
+        Splatterwarshrade();
+        Deadfly();
+        Oddities();
+    }
+
+    public void Skullpunch()
+    {
+        // Check progress and membership
```

Comment:
this is fine but in the future there no need for comments like //check progress and // preload already code is understandable no need for comments for it